examples/plotting/ImageAnimatorWCS_Example.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
Oh also, I think we can rename the file to something else. make it all lower case and drop the `_example` bit.